tutors3/patches/openedx-common-settings

```diff
@@ -15,7 +15,7 @@ AWS_S3_ENDPOINT_URL = "{{ "https" if S3_USE_SSL else "http" }}://{{ S3_HOST }}{%
 
 AWS_S3_USE_SSL = {{ "True" if S3_USE_SSL else "False" }}
 AWS_S3_SECURE_URLS = {{ "True" if S3_USE_SSL else "False" }}
-AWS_DEFAULT_ACL = None # inherit from the bucket
+AWS_DEFAULT_ACL = {{ '"{}"'.format(S3_DEFAULT_ACL) if S3_DEFAULT_ACL is defined else 'None' }} # None - inherit from the bucket
```

Comment:
We tend to not use `if x is defined` constructs in our plugins. Instead, we define a default in `plugin.py` (even if that default is None).

Reply:
Cool. Looking forward merging #83